toolkit/themes/shared/design-system/build.js

```diff
@@ -0,0 +1,12 @@
+const StyleDictionary = require('style-dictionary');
+
+// APPLY THE CONFIGURATION
+// IMPORTANT: the registration of custom transforms
+// needs to be done _before_ applying the configuration
+const StyleDictionaryExtended = StyleDictionary.extend(__dirname + '/config.js');
+
+// FINALLY, BUILD ALL THE PLATFORMS
+StyleDictionaryExtended.buildAllPlatforms();
+
+console.log('\n==============================================');
+console.log('\nBuild completed!');
```

Comment:
now that things are in `config.js` we don't need this file at all anymore - you can just delete it

Reply:
@hannajones but then how do we run the build? I tried `node config.js` and nothing would happen lol. What am I missing?

Reply:
I just run `style-dictionary build` from the command line and don't bother with node 🤷🏻 If you want you can add something like this to `package.json` to make it work with `npm run build`:

```json
"scripts": {
  "build": "style-dictionary build"
}
```

Reply:
oh duh, I should have been using `style-dictionary` to build it; totally forgot about it for some reason. Thanks!